Alignment/OfflineValidation/test/submitAllJobs.py

```diff
@@ -0,0 +1,817 @@
+
+            #if(to_bool(isMC[iConf]) and (not to_bool(doRunBased))):
+            if(to_bool(isMC[iConf])):
+                thejobIndex=jobN
+            else:
+                thejobIndex=myRuns[jobN]
+
+            aJob = Job(thejobIndex,
+                       jobName[iConf],isDA[iConf],isMC[iConf],
+                       applyBOWS[iConf],applyEXTRACOND[iConf],conditions[iConf],
+                       myRuns[jobN], lumilist[iConf], maxevents[iConf],
+                       gt[iConf],allFromGT[iConf],
+                       alignmentDB[iConf], alignmentTAG[iConf],
+                       apeDB[iConf], apeTAG[iConf],
+                       bowDB[iConf], bowTAG[iConf],
+                       vertextype[iConf], tracktype[iConf],
+                       applyruncontrol[iConf],
+                       ptcut[iConf],input_CMSSW_BASE,AnalysisStep_dir)
+            
+            aJob.setEOSout(eosdir)
+            aJob.createTheCfgFile(theSrcFiles)
+            aJob.createTheLSFFile()
+
+            output_file_list1.append("cmsStage "+aJob.getOutputFileName()+" /tmp/$USER/"+opts.taskname+" \n")
+            if jobN == 0:
+                output_file_list2.append("/tmp/$USER/"+opts.taskname+"/"+aJob.getOutputBaseName()+".root ")
+            output_file_list2.append("/tmp/$USER/"+opts.taskname+"/"+os.path.split(aJob.getOutputFileName())[1]+" ")    
+   
+            if opts.submit:
+                aJob.submit()
+                batchJobIds.append(ajob.getBatchjobId())
+            del aJob
+
+        if opts.submit:
+            print "********************************************************"
+            for theBatchJobId in batchJobIds:
+                print "theBatchJobId is: ",theBatchJobId
+
+        fout.write("#!/bin/bash \n")
+        fout.write("MAIL = $USER@mail.cern.ch \n")
+        fout.write("OUT_DIR = "+eosdir+ "\n")
+        fout.write("echo $HOST | mail -s \"Harvesting job started\" $USER@mail.cern.ch \n")
+        fout.write("cd "+os.path.join(input_CMSSW_BASE,"src")+"\n")
+        fout.write("eval `scram r -sh` \n")
+        fout.write("mkdir -p /tmp/$USER/"+opts.taskname+" \n")
+        fout.writelines(output_file_list1)
+        fout.writelines(output_file_list2)
+        fout.write("cmsStage -f $FILE $OUT_DIR \n")
+        fout.write("echo \"Harvesting for complete; please find output at $OUT_DIR \" | mail -s \"Harvesting for" +opts.taskname +" compled\" $MAIL \n")
+
+        os.system("chmod u+x "+hadd_script_file)
+
+        conditions = '"' + " && ".join(["ended(" + jobId + ")" for jobId in batchJobIds]) + '"'
+        print conditions
+        lastJobCommand = "bsub -o harvester"+opts.taskname+".tmp -q 1nh -w "+conditions+" "+hadd_script_file
+        print lastJobCommand
+        if opts.submit:
+            lastJobOutput = getCommandOutput(lastJobCommand)
+            print lastJobOutput
+
+        fout.close()
+        del output_file_list1
+        
+if __name__ == "__main__":        
+    main()
+
+
+   
```

Comment:
@mmusich looks a bit like duplication of the all-in-one tool. is this meant to be used stand-alone?